src/pages/tokenDrop/Payment.tsx

```diff
@@ -49,12 +49,9 @@ const Payment: FunctionComponent = () => {
   const [timeRemaining, setTimeRemaining] = useState("--:--");
 
   const paymentAddress = purchaseOrder?.paymentAddress;
-  const activePurchase =
-    !!purchaseStatus &&
-    [PurchaseStatus.Pending, PurchaseStatus.Processing].includes(
-      purchaseStatus
-    );
-
+  const isPending = purchaseStatus === PurchaseStatus.Pending;
+  const isProcessing = purchaseStatus === PurchaseStatus.Processing;
+  const activePurchase = !!purchaseStatus && (isPending || isProcessing);
   const isSupportedBrowser = ["Chrome", "Brave"].includes(browserName);
```

Comment:
They can still use other browsers if they're doing the entirely manual flow, correct?